src/etc/rust_types.py

```diff
@@ -111,7 +111,7 @@ def classify_struct(name, fields):
     return RustType.STRUCT
 
 
-def classify_union(fields):
+def classify_union(fields: List) -> str:
```

Comment:
Can the `List` type be narrowed in this file? Per https://lldb.llvm.org/python_api/lldb.SBType.html#lldb.SBType.fields I think this could be `List[SBTypeMember]`.

Reply:
In this file they can't be, as the functions are shared between LLDB and GDB.